src/azure-cli/azure/cli/command_modules/policyinsights/tests/latest/test_policyinsights_scenario.py

```diff
@@ -486,22 +496,379 @@ def test_policy_insights_remediation_reevaluate(self, resource_group_location, s
     @AllowLargeResponse(8192)
     def test_policy_insights_metadata(self):
         # Get all metadata resources
-        all_metadata_resources = self.cmd('policy metadata list').get_output_in_json()
+        all_metadata_resources = self.cmd(
+            'policy metadata list').get_output_in_json()
         assert len(all_metadata_resources) > 1
 
         # Test the --top argument
-        assert len(self.cmd('policy metadata list --top 0').get_output_in_json()) == 0
+        assert len(
+            self.cmd('policy metadata list --top 0').get_output_in_json()) == 0
 
-        top_metadata_resources = self.cmd('policy metadata list --top {}'.format(len(all_metadata_resources) + 1)).get_output_in_json()
+        top_metadata_resources = self.cmd(
+            'policy metadata list --top {}'.format(len(all_metadata_resources) + 1)).get_output_in_json()
         assert len(top_metadata_resources) == len(all_metadata_resources)
 
-        top_metadata_resources = self.cmd('policy metadata list --top {}'.format(len(all_metadata_resources) - 1)).get_output_in_json()
+        top_metadata_resources = self.cmd(
+            'policy metadata list --top {}'.format(len(all_metadata_resources) - 1)).get_output_in_json()
         assert len(top_metadata_resources) == len(all_metadata_resources) - 1
 
         # Test getting an individual resouce
         resource_metadata_name = top_metadata_resources[0]['name']
-        metadata_resource = self.cmd('policy metadata show --name {}'.format(resource_metadata_name)).get_output_in_json()
+        metadata_resource = self.cmd(
+            'policy metadata show --name {}'.format(resource_metadata_name)).get_output_in_json()
         assert metadata_resource['name'] == resource_metadata_name
 
-        metadata_resource = self.cmd('policy metadata show -n {}'.format(resource_metadata_name)).get_output_in_json()
+        metadata_resource = self.cmd(
+            'policy metadata show -n {}'.format(resource_metadata_name)).get_output_in_json()
         assert metadata_resource['name'] == resource_metadata_name
+
+    # Test policy attestation CRUD requests at various scopes
+    @record_only()
```

Comment:
Why is this test marked as `@record_only()`?

Reply:
Policies take a variable amount of time to evaluate compliance states for applicable resources and is time-intensive to setup for a live test. And that is why I had that test to be record_only. Will add the comments and steps to re-record these tests.

Reply:
Got it, thanks~